ote/resources/public/language/fi.edn

```diff
@@ -124,6 +124,39 @@
   ;; Parking fields
   :parking
   {:parking-areas  "Parkkipaikkojen sijainnit"
+   :ote.db.transport-service/luggage-restrictions      "Matkatavaroita koskevat rajoitukset"
```

Comment:
hmm.. Tarviikohan meidän kantaa huolta siitä, että parkkipaikoilla ei saisikaan jotain matkatavaroita kuljettaa?

Reply:
Accessibility issues will be addressed in another ticker.